shared/Home/UseCases.tsx

```diff
@@ -106,45 +134,66 @@ export const engine = inngest.createFunction(
 
 const content = [
   {
-    title: "Durable workflows",
+    title: "LLM Chains",
     content: (
       <p>
-        Combine any complex series of tasks into a single reliable workflow.
-        Each task becomes a “step” which is automatically retried when errors
-        happen.
+        Get your LLM apps running in production without the complexity of glue
+        code and additional infrastructure.
       </p>
     ),
     bullets: [
-      <>Run steps in parallel or series or call a child workflow.</>,
+      <>Handle complex text-generation with chain-based post-processing.</>,
       <>
-        Add durable sleep to pause your workflow for days or weeks at a time.
+        Leverage Retrieval Augmented Generation (RAG) by querying vector stores
+        without complex interfaces and building ingestion functions.
       </>,
+      <>Wrap steps to run exactly once to reduce extra, expensive API calls.</>,
+      <>Limit concurrency and prioritize jobs ahead of others</>,
+    ],
+    snippet: snippetAI,
+    href: "/ai?ref=homepage",
+  },
+  {
+    title: "AI Agents",
+    content: (
+      <p>
+        Build long-running autonomous agent functions that make decisions and
+        call other functions to complete complex tasks.
+      </p>
+    ),
+    bullets: [
       <>
-        Visually debug the entire workflow without having to parse logs and
-        connect the dots.
+        Define human-in-the-loop flows that wait for approval or additional
+        input.
       </>,
+      <>Create functions that track state across actions</>,
+      <>Define actions as normal functions with built-in retries</>,
+      <>Build a system of multiple agents that work together</>,
```

Comment:
```suggestion
      <>Build a system of multiple agents that interact with each other.</>,
```

I'm not sure if they "work" together.

Reply:
Definitely inspired by the idea that agents actually perform "work." "Collaborate" is another words I've seen used.

Reply:
You're right ✅

Reply:
Combined ideas 👍 😄